src/twisted/persisted/_token.py

```diff
@@ -0,0 +1,146 @@
+"""
+Token constants.
+vendored from https://github.com/python/cpython/blob/6b825c1b8a14460641ca6f1647d83005c68199aa/Lib/token.py
+Licence: https://docs.python.org/3/license.html
+"""
```

Comment:
just a warning

```suggestion
"""
FIXME:https://github.com/twisted/twisted/issues/3843
This can be removed once t.persisted.aot is removed.
New code should not make use of this.

Token constants.
vendored from https://github.com/python/cpython/blob/6b825c1b8a14460641ca6f1647d83005c68199aa/Lib/token.py
Licence: https://docs.python.org/3/license.html
"""
```